ethminer/main.cpp

```diff
@@ -49,6 +49,7 @@ void help()
 		<< "    -h,--help  Show this help message and exit." << endl
 		<< " Envionment variables:" << endl
 		<< "     NO_COLOR - set to any value to disable color output. Unset to re-enable color output." << endl
+		<< "     SYSLOG   - set to any value to strip time and disable color from output, for logging under systemd" << endl
```

Comment:
Why not regular command line option like `--syslog`?

Reply:
I made it like "NO_COLOR" since it's a "superset" of the NO_COLOR behaviour -- removes color AND strips timestamp.  
Also, since environment variables are easily set in systemd unit files, it can be set there to affect the output only when started via systemd.  Then you don't have to edit startup wrapper script if you want to run with regular output.